roles/benchmarking_deploy_coco_dataset/files/entrypoint.sh

```diff
@@ -5,107 +5,19 @@ set -u
-e52f412dd7195ac8f98d782b44c6dd30ea10241e9f42521f67610fbe055a74f8  image_info_test2017.zip
-EOF
-
-declare -A DIR_PREFIXES=(
-    ["test2017.zip"]="zips"
-    ["train2017.zip"]="zips"
-    ["val2017.zip"]="zips"
-    ["annotations_trainval2017.zip"]="annotations"
-    ["image_info_test2017.zip"]="annotations"
-)
-
-if [[ -z "$CERT_FILE" ]]; then
-    echo "INFO: no cert file provided, downloading from upstream."
-
-    DATASET_BASE_URL="http://images.cocodataset.org"
-    echo "Using the upstream base URL: $DATASET_BASE_URL"
-
-elif [[ ! -e "$CERT_FILE" ]]; then
-    echo "FATAL: cert file not found ($CERT_FILE)"
-    exit 1
-fi
-
 echo "Installing 'unzip' ..."
 dnf install unzip -y --quiet
 
-for dataset_filename in $(cat checksum.sha256sum | cut -d' ' -f3);
-do
-    if [[ -f "$dataset_filename".extracted ]]; then
-        echo "$dataset_filename already downloaded and extracted, skipping."
-        continue
-    fi
+curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+unzip awscliv2.zip
+./aws/install
+/usr/local/bin/aws configure import --csv file://${CERT_FILE}
 
-    grep "  $dataset_filename" checksum.sha256sum | sed "s/$dataset_filename/-/" > stdin.sha256sum
-
-    if [[ -e "$dataset_filename" ]]; then
-        echo "INFO: Found an existing '$dataset_filename'. Checking its sha256sum ..."
-        ls -lh "$dataset_filename"
-        if ! cat "$dataset_filename" | sha256sum --check "stdin.sha256sum"; then
-            echo "INFO: '$dataset_filename' check sum is invalid. Delete it."
-            ls -lh "$dataset_filename"
-            rm "$dataset_filename"
-        else
-            echo "INFO: $dataset_filename already available."
-            continue
-        fi
-    fi
-
-    url="$DATASET_BASE_URL"
-    if [[ -z "$CERT_FILE" ]]; then
-        url="${url}/${DIR_PREFIXES[$dataset_filename]}"
-        cert=""
-    else
-        cert="--cert $CERT_FILE"
-    fi
-    url="${url}/$dataset_filename"
-
-    echo "Downloading $url ..."
-    touch "${dataset_filename}.url"
-
-    if ! time \
-         curl \
-         $cert \
-         --silent  --fail --show-error \
-         --retry 999999 \
-         --retry-max-time $(($DOWNLOAD_MAX_TIME_MIN * 60)) \
-         --continue-at - \
-         "${url}" \
-            | tee "$dataset_filename" \
-            | sha256sum --check stdin.sha256sum;
-    then
-        echo "FATAL: failed to download/verify $dataset_filename ..."
-        exit 1
-    fi
-done
-
-for dataset_filename in $(cat checksum.sha256sum | cut -d' ' -f3);
-do
-    if [[ -f "$dataset_filename".extracted ]]; then
-        echo "$dataset_filename already extracted."
-        continue
-    fi
-    echo "Extracting $dataset_filename ..."
-    unzip "$dataset_filename" >/dev/null
-
-    touch "${dataset_filename}.extracted"
-    echo "Deleting $dataset_filename ..."
-    rm -f "$dataset_filename"
-done
+/usr/local/bin/aws s3 cp s3://psapcocobucket/ /storage --profile=cocobucket --recursive
```

Comment:
can you replace `/storage` by an env variable, that you defined in the Pod template? (eg, `STORAGE_DIR`)
(will make it easier to reuse the script, as when Anisha wanted to download the dataset on RHEL)